package.json

```diff
@@ -25,14 +26,14 @@
     "eslint": "^4.1.1",
     "eslint-config-ttn": "^1.3.2",
     "flow-bin": "^0.55.0",
+    "flow-typed": "^2.2.3",
     "jest": "^20.0.4",
     "jest-cli": "^20.0.4",
     "jest-preset-ttn": "^0.0.3",
     "json": "^9.0.6"
   },
   "dependencies": {
     "debug": "^2.6.8",
-    "flow-typed": "^2.1.5",
```

Comment:
Why remove this from `dependencies`? Isn't it needed for a local `npm install`?

Reply:
A `npm install` in this repo will still install the dev dependencies, so everything should be fine for development.

An application depending on `ttn` will not install `ttn`s devDependendencies.
`flow-typed` is not needed at runtime, so I considered devDependencies a better place for it

Reply:
> flow-typed is not needed at runtime,
No, but is it needed in the `prepare` step? (My knowledge of npm hooks is very limited).

Reply:
The `prepare` hook is executed
- when running `npm install` locally, but not when installed as dependency
- when publishing to npm

It's a workaround to run the hook only on *local* `npm install`, b/c there is no better hook available.
If you don't consider this is not appropriate (i'm not shure either), let's add a note to the readme, hinting to run `make flow-deps` before typechecking.

Another idea would be to add a make dependency for `make typecheck`. It would run each time though (?), slowing the typecheck down

Reply:
No the prepare hook seems fine!

Reply:
Can you resolve the conficts? Then we can merge!

Reply:
done, thanks!
tip: latest yarn can automatically solve merge conflicts in yarn.lock by running `yarn`